flutter/Graminsta/lib/auth.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import 'package:Graminsta/login.dart';
+
+class AuthPage extends StatefulWidget {
+  AuthPage({Key key}) : super(key: key);
+
+  @override
+  _AuthPageState createState() => new _AuthPageState();
```

Comment:
Not sure if I'm right, please double check. As part of the Dart 2, `new` is optional. and Effective Dart suggests not to use that.

See also: https://dart.dev/guides/language/effective-dart/usage#dont-use-new

Reply:
Ditto for all.